.openpublishing.redirection.json

```diff
@@ -11631,6 +11631,10 @@
             "source_path": "docs/standard/serialization/add-element-for-xmlschemaimporterextensions.md",
             "redirect_url": "/dotnet/standard/serialization/add-element-for-schemaimporterextensions"
         },
+        {
+            "source_path": "docs/standard/serialization/basic-serialization-technology-sample.md",
```

Comment:
This serialization sample docs page only gets about 100 page views/month. The link to download the sample is broken and I can't find a replacement, so I deleted the article.